src/main/java/stream/flarebot/flarebot/util/MessageUtils.java

```diff
@@ -104,7 +104,7 @@ public static Message sendException(String s, Throwable e, MessageChannel channe
     public static String paste(String trace) {
         if (FlareBot.getInstance().getPasteKey() == null || FlareBot.getInstance().getPasteKey().isEmpty()) {
             FlareBot.LOGGER.warn("Paste server key is missing! Pastes will not work!");
-            return;
+            return null;
```

Comment:
Could cause potential issues by returning null. I would recommend returning an empty string

Reply:
We already do this when hastebin has issues, we never actually use the string just pass it therefor this will still be correct and will appear as "null" in the error log rather than nothing appearing.

Reply:
Ah ok